### ISSUE TITLE
Watcher is not working on Windows

### ISSUE THREAD

**Issue by amstiel** [stepanzin/gulpmeister#13]
Watcher is starts, but it's not runs task after file changes. 

First I thought that reason is in `gulp.watch` and `chokidar`. Found some old issues on github about it.

But after some research I realize that `gulp.watch` accepts `glob` path format, and `path.join` on Windows returns "Windows" path format ('\' and so on).

So, when I changed `path.join(srcPath, '/**/*.{scss, sass}')` to `srcPath + '/**/*.{scss, sass}'` Watcher started run task after file changes.

In conclusion, **reason** why Watcher is not runs tasks is because it not see changes because path of file is incorrect.

Possible **solutions**: not use `path.join` for Watcher or change `path.join` behavior on Windows.

**Comment by Grawl** [comment]
for windows backslach support I recomment you to not use any slashes at all.

Intead of this:

```js
const path = path.join(srcPath, '/**/*.{scss, sass}')
```

Try to create path like this:

```js
const path = path.join(srcPath, '**', '*.{scss, sass}')
```

Maybe you can find some way to convert slashes for platform.

**Comment by amstiel** [comment]
@Grawl you don't got it. `path.join` returns valid Windows path, like
```js
const srcPath = './themes/frontend';
const path = path.join(srcPath, '/**/*.{scss, sass}'); 
// returns "themes\frontend\**\*.{scss, sass}
```
This is **exactly** the problem. `gulp.watch` (and `chokidar`, under the hood of this method) working with path like `./themes/frontend/bla/bla`, regardless of the OS on which it runs.

**Comment by amstiel** [comment]
I found possible solution. We can import path with specific method implementations:
```js
const path = require('path').posix; // or .win32
```
So `join` and other methods will return paths in specific format, regardless of the environment;

**Comment by Grawl** [comment]
Good solution
